DeveloperLabs/MyEmployees/MyEmployees.Updater/ComUpdater/BackgroundUpdateRegister.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+// Required for BackgroundTaskBuilder APIs.
+using Windows.ApplicationModel.Background;
+
+// Required for COM registration API (RegisterTypeForComClients).
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+
+namespace MyEmployeesUpdater.ComUpdater
+{
+    public class BackgroundUpdateRegister
+    {
+
+        /// <summary>
+        /// Register a background task with the specified name and trigger
+        /// </summary>
+        /// <param name="name">A name for the background task.</param>
+        /// <param name="trigger">The trigger for the background task.</param>
+        public static void RegisterBackgroundTaskWithSystem(string taskName, IBackgroundTrigger trigger)
+        {
+            // If the task is already active, the function will be returned here rather than registering another instance of it
+            foreach (var regIterator in BackgroundTaskRegistration.AllTasks)
+            {
+                if (regIterator.Value.Name == taskName)
+                {
+                    return;
+                }
+            }
+
+            // Build an instance of the task with taskEntrypoint, name, and trigger
+            BackgroundTaskBuilder builder = new BackgroundTaskBuilder();
+            builder.SetTrigger(trigger);
+            builder.Name = taskName;
+            builder.SetTaskEntryPointClsid(typeof(ComBackgroundUpdate).GUID);
+
+            // Register the task if it has not been registered
+            BackgroundTaskRegistration registration;
+            try
+            {
+            registration = builder.Register();
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e.Message);
+                registration = null;
+                return;
+            }
+            RegisterProcessForBackgroundTask(typeof(ComBackgroundUpdate));
+        }
+
+        /// <summary>
```

Comment:
Delete this line and line 65 `/// </summar>`

/// Is the doxygen doc-comment indicator and VS understands (since VS2019?). The 'summary' tags are just noise here.